.info

```diff
@@ -0,0 +1 @@
+check if karpenter removes daemonsets during draining or if it checks if daemonsets r there when its marks the node to be empty
```

Comment:
I'd add this file to your `.gitignore`. I don't think that this file should have been added

Reply:
sorry ill add that